src/screens/user/MyJobScreen.tsx

```diff
@@ -0,0 +1,373 @@
+import React, { memo, useState } from 'react';
+import Background from '../../components/Background';
+import Container from '../../components/Container';
+import Button from '../../components/Button';
+import { Navigation } from '../../types';
+import { connect } from 'react-redux';
+import {
+  Modal,
+  View,
+  StyleSheet,
+  Alert,
+  Linking,
+  ScrollView,
+} from 'react-native';
+import { MaterialIcons } from '@expo/vector-icons';
+import TextInput from '../../components/TextInput';
+import {
+  titleValidator,
+  descriptionValidator,
+  priceValidator,
+  cityValidator,
+  postalCodeValidator,
+  addressValidator,
+} from '../../core/utils';
+import * as ImagePicker from 'expo-image-picker';
+import * as Permissions from 'expo-permissions';
+import Constants from 'expo-constants';
+import { theme } from '../../core/theme';
+import gql from 'graphql-tag';
+import { useMutation, useQuery } from '@apollo/react-hooks';
+import { jobToGlobalState } from '../../store/user/actions';
+import { Avatar, Card, Title, Paragraph } from 'react-native-paper';
+
+type Props = {
+  navigation: Navigation;
+  user: { country: String; id: String; jobs: [] };
+  jobToGlobalState: (job: Object) => void;
+};
+
+//TODO: ADD type for images $images: [Upload!]! || single file not required: Upload
+const ADD_JOB_MUTATION = gql`
```

Comment:
I'd move all mutations to other files. Maybe one with all the mutations or maybe you can extend the structure you already have in your redux store and add a mutation file per reducer. Just `export const ADD_JOB_MUTATION = gql'..'` and use it where you need it

Reply:
done